src/components/NcAppContent/NcAppDetailsToggle.vue

```diff
@@ -46,6 +51,9 @@ export default {
 		title() {
 			return t('Go back to the list')
 		},
+		isRTL() {
+			return isRTL(getLanguage())
```

Comment:
BTW, `isRTL` already has a fallback `getLanguage()`
https://github.com/nextcloud-libraries/nextcloud-l10n/blob/03d07c211f7381e927955ce9fe899cdd5ccc2a26/lib/locale.ts#L33-L34

Reply:
```suggestion
			return isRTL()
```

Reply:
But this does not depend on any runtime variable, so please move this to `setup` instead @hamza221